source/features/action-used-by-link.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+export const isActionPage = (): boolean => location.pathname.split('/', 3).join('/') === '/marketplace/actions';
```

Comment:
I suppose there's some utility functions that I didn't understand that'd improve this? :grimacing:

Reply:
Have a look at `page-detect.ts` and match their style. Include tests

Reply:
Should the function live in page-detect.ts or in this feature file?

Reply:
Could this test be useful for other features?

In this case the answer is yes, so it belongs to `page-detect.ts`